amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/InterchangeEndpoints.java

```diff
@@ -238,6 +241,99 @@ public Response getValues(List<String> fields) {
         }
         return Response.ok(response, responseType).build();
     }
+    
+    /**
+     * Returns a list of values for all id of requested fields.
+     * 
+     * For fields like locations, sectors, programs the object contains the full (ancestor) path.
+     * <h3>Sample request:</h3><pre>
+     * [
+     *   "locations~location": [534, 126],
+     *   "national_plan_objective~program": [123],
+     *   "primary_sectors~sector": [297]
+     * ]
+     * </pre>
+     * <h3>Sample response:</h3><pre>
+     * {
+     *   "locations~location": [
+     *     {
+     *       "id": 534,
+     *       "value": "2ème Section Bois Neuf",
+     *       "ancestor-path": "[Haiti] [Artibonite] [Gros Morne Arrondissement] [Terre-Neuve] [2ème Section Bois Neuf]"
```

Comment:
Let's provide `"ancestor-values": ["Haiti", ..., "2ème Section Bois Neuf"]` for FE flexibility on formatting